resources/qml/dialogs/HiddenEventsDialog.qml

```diff
@@ -17,6 +17,8 @@ ApplicationWindow {
 
     modality: Qt.NonModal
     flags: Qt.Dialog | Qt.WindowTitleHint
+    width: 275
+    height: 220
```

Comment:
Why do you explicitly need to set a size here?

Reply:
Otherwise it stretches and has a lot of empty space, which bothers me. :D

Reply:
Weird, why would it do that? It never did for me :D

Reply:
IDK. But anyway, there are other dialogs where we set an explicit size as well, so I don't think this is too unusual to do.

Reply:
Well, it is weird that we don't set it to the minimum size, I guess?

Reply:
I mean, yes, but you are making it specifically 25px wider, why?

Reply:
Because that kept the top label from taking two lines for the title "nheko". I could make it larger to help with slightly longer titles.

Reply:
Hm, okay, I think that is fine then, ty.